ignite/cmd/network_chain_prepare.go

```diff
@@ -112,7 +113,11 @@ func networkChainPrepareHandler(cmd *cobra.Command, args []string) error {
 	session.StopSpinner()
 	session.Printf("%s Chain is prepared for launch\n", icons.OK)
 	session.Println("\nYou can start your node by running the following command:")
-	commandStr := fmt.Sprintf("%s start --home %s", binaryName, chainHome)
+	startCmd := "start"
+	if gitpod.IsOnGitpod() {
+		startCmd = "start-with-tunneling"
+	}
+	commandStr := fmt.Sprintf("%s %s --home %s", binaryName, startCmd, chainHome)
```

Comment:
As discussed, a new command has been added : `appd start-with-tunneling`, which is basically the same as `start` but with http tunneling from spn config.

Here the `network prepare` output suggest to use `start-with-tunneling` instead of `start` if the host is gitpod.